monai/transforms/post/array.py

```diff
@@ -248,3 +248,37 @@ def __call__(self, img):
             output = img
 
         return output
+
+
+class Contour(Transform):
+    """
+    Return the contour of organs in one mask image that only composes of 0 and 255 in gray value,
+    with Laplace kernel set as default for edge detection.
+
+    Args:
+        kernel_type: the method applied to do edge detection.
+    """
+
+    def __init__(self, kernel_type='Laplace'):
+        self.kernel_type = kernel_type
+
+    def __call__(self, img):
+        """
+        Args:
+            img: torch tensor of the img that you want to find the contour of, with shape being (1, 1, width, height)
+
+        Returns:
+            A torch tensor representing contour image, note:
+                1. it's gray image, rather than binary classification result of whether edge or not.
+                2. In order to keep the original shape of mask image, we use padding as default.
+        """
+        if self.kernel_type != 'Laplace':
+            raise NotImplementedError
+
+        kernel = torch.tensor([[-1, -1, -1], [-1, 8, -1], [-1, -1, -1]], dtype=torch.float32).view((1, 1, 3, 3), device=img.device)
+        conv = torch.nn.Conv2d(1, 1, kernel_size=3, stride=1, padding=1, bias=False)
```

Comment:
Hi @wuyuanhang94 ,

Could you please help support both 2D and 3D input data?
Thanks.